### PR TITLE
Add togeojson via package.json w/ npm auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10425
Related issue(s): # #

@cdnjs/intern3 Please help me on this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mapbox/togeojson
 * Official website (optional, not the repository): http://mapbox.github.io/togeojson/
 * NPM package url (optional): https://www.npmjs.com/package/togeojson
 * License and its reference: BSD-2-Clause, ref: https://github.com/mapbox/togeojson/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 119
   - Count of stars: 579
   - Count of forks: 134
 * NPM download stats (optional):
   - Downloads in the last day: 37
   - Downloads in the last week: 1114
   - Downloads in the last month: 4256

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/togeojson/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "maps"
+  ],
+  "homepage": "https://mapbox.github.io/togeojson/",
+  "author": "tmcw",
+  "npmName": "togeojson",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "togeojson.js"
```
Comment (clairetsai818) [inline ajax/libs/togeojson/package.json]: Maybe using `togeojson*.(map|js|css)` will make the filemap more flexible.

Comment (hare1039) [inline ajax/libs/togeojson/package.json]: I can't fetch any file using this rule.

Comment (clairetsai818) [inline ajax/libs/togeojson/package.json]: sorry, I think it's `togeojson*.+(map|js|css)`

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by hare1039** [comment]
@cdnjs/intern3 Sorry for late update. Had a busy week...
I update the rules, please review this PR again. Thanks.

File: ajax/libs/togeojson/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "gpx",
+    "geo",
+    "parser",
+    "formatter",
+    "input",
+    "leaflet",
+    "maps"
+  ],
+  "homepage": "https://mapbox.github.io/togeojson/",
+  "author": "tmcw",
```
Comment (PeterDaveHello) [inline ajax/libs/togeojson/package.json]: Where is the author info come from?

Comment (hare1039) [inline ajax/libs/togeojson/package.json]: He is the main contributor of the library https://github.com/mapbox/togeojson/graphs/contributors

Comment (PeterDaveHello) [inline ajax/libs/togeojson/package.json]: Then you should use "Tom MacWright (http://macwright.org/)" ...

**Comment by hare1039** [comment]
@pvnr0082t I fix the author field. Please review this PR again. Thanks.

**Pull request merged**